src/main.rs

```diff
@@ -162,8 +162,8 @@ fn main() {
         file.write_all(
             r#"{
     "useragent": "ftb-rs",
-    "username": "insert username here",
-    "password": "insert password here",
+    "username": "insert bot username here",
```

Comment:
We should probably rename these fields or at least clarify them since it isn't a real username. I use the word "login" for ESAEBSAD2. For password it can just be called "token" because that's what it is.